app/code/Codifi/Sales/etc/di.xml

```diff
@@ -9,7 +9,10 @@
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <type name="Magento\Sales\Api\OrderRepositoryInterface">
-        <plugin name="order_type_add_order_extension_attribute" type="Codifi\Sales\Plugin\OrderRepositoryPlugin" />
+        <plugin name="orderTypeAddOrderExtensionAttribute" type="Codifi\Sales\Plugin\Api\OrderRepositoryInterfacePlugin" />
+    </type>
+    <type name="Magento\Quote\Api\CartRepositoryInterface">
+        <plugin name="orderTypeAddQuoteExtensionAttribute" type="Codifi\Sales\Plugin\Api\CartRepositoryInterfacePlugin" />
```

Comment:
1. It's better to declare module name in path to plugin:
Codifi\Sales\Plugin\ **Quote** \Api\CartRepositoryInterfacePlugin
2. It will be more understandable: `AddOrderTypeToQuoteExtensionAttributes`